playbooks/group_vars/promtail_servers/00-defaults.yaml

```
@@ -1,15 +1,33 @@
 # vim:ts=2:sw=2:et:ai:sts=2
 ---
 
-apt__packages_to_install__promtail:
-  - promtail
-apt__services_to_enable__promtail:
-  - promtail
-apt__services_to_start__promtail:
-  - promtail
-
-ferm__services__promtail:
-  promtail:
-    dport: 9080
-    proto: tcp
-    saddr: '{{ internal_networks }}'
+promtail__listen_port: 9080
+promtail__storage_dir: /var/lib/promtail
+
+# If true, install ferm configuration file.
+promtail__install_ferm_svc: '{{ ferm__enable | bool }}'
+# Optionally limit access to a list of IP addresses.
+promtail__ferm_allow_from: '{{ internal_networks }}'
+
+# List of Loki instances to push logs to.
+promtail__client_urls: []
+# Parameters for Loki clients.
+promtail__client_defaults: {}
+# Full push clients configuration block,
+# if set overrides `promtail__client_urls` and `promtail__client_defaults`.
+promtail__clients: |
+  {% set res = [] %}
+  {% for url in promtail__client_urls %}
+  {% set _ = res.append(promtail__client_defaults | combine(dict(url=url))) %}
+  {% endfor %}
+  {{- res -}}
+
+promtail__scrape_configs: |-
+  {{ query('vars', *(query('varnames', '^promtail__scrape_config__.*')
+     | select | sort)) }}
+promtail__scrape_config__varlogs:
+  job_name: varlogs
+  static_configs:
+    - labels:
+        job: varlogs
+        __path__: /var/log/*log
```

This glob will match `/var/log/lastlog`, which is not in plaintext format. Should it be:

```suggestion
        __path__: /var/log/*.log
```

?